docs/geedocs/5.2.5/answer/176738.html

```diff
@@ -273,6 +273,10 @@ <h2 id="cache_configuration">Cache configuration</h2>
 <br><br>
 # the size of the allocation blocks<br>
 BLOCK_SIZE=&lt;value&gt;
+<br><br>
+# whether or not purge the xml cache
+# false = 0, true = 1
+PURGE = &lt;value&gt; 
 </code>
```

Comment:
The PERCENT option is missing here.

Reply:
Bump - still a missing option in the documentation. It was added to the 5.3.0 docs but not 5.2.5.